Tests/SwiftParserTest/translated/InvalidTests.swift

```diff
@@ -305,14 +305,17 @@ final class InvalidTests: XCTestCase {
   }
 
   func testInvalid13() {
+    // TODO: Determine if the comment below should be removed.
     // https://github.com/apple/swift/issues/43190
     // Crash with invalid parameter declaration
+    
+    // TODO: Remove the trailing trivia from the firstName node (`s` and space).
```

Comment:
This might actually resolve itself if you apply my other comment above and if you don’t need special handling in `ParseDiagnosticGenerator`.